rticonnextdds_connector/rticonnextdds_connector.py

```diff
@@ -679,7 +730,7 @@ def __init__(self, connector, name):
 		self.name = name
 		self.native = connector_binding.getReader(self.connector.native,tocstring(self.name))
 		_check_entity_creation(self.native, "Input")
-		self.samples = Samples(self)
+		self._samples = Samples(self)
```

Comment:
For consistency I would use the same syntax for samples and infos (i.e., both with a preceeding _ or neither with it)

Reply:
It needs the _ because I provide it through a property. So it can't have the same name, and the attribute must be private

Reply:
The infos is for backward compatibility and it is accessible.

_samples is accessible thru the property